neurons/miner.py

```diff
@@ -174,7 +181,7 @@ async def submit_model(self) -> None:
 
         # Push model metadata to chain
         model_id = ChainMinerModel(
-            competition_id=self.config.competition.id,
+            competition_id=self.config.competition_id,
             hf_repo_id=self.config.hf_repo_id,
             hf_model_filename=self.config.hf_model_name,
             hf_repo_type=self.config.hf_repo_type,
```

Comment:
jeżeli robimy tylko repo typu "model", to z tych miejsc też można wywalić repo_type